jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/Dialog.java

```diff
@@ -0,0 +1,103 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.DialogAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+
+/**
+ * To see an example of Dialog web element please visit https://material.angular.io/components/dialog/overview.
+ */
+
+public class Dialog extends UIBaseElement<DialogAssert> {
+    private Button dialog;
+    private String answerLabel = "dialog-overview-example li.ng-star-inserted";
```

Comment:
All of these fields must be public and not private.

Reply:
Fixed, thanks.